build.rs

```diff
@@ -145,6 +145,7 @@ fn cross_compiling() -> Result<Option<CrossCompileConfig>> {
     if env::var_os("PYO3_CROSS").is_none()
         && env::var_os("PYO3_CROSS_LIB_DIR").is_none()
         && env::var_os("PYO3_CROSS_PYTHON_VERSION").is_none()
+        && env::var_os("PYO3_CROSS_NO_ENABLE_SHARED").is_none()
```

Comment:
I think this also needs to be added here:
https://github.com/PyO3/pyo3/blob/c0988f62d14a702abc64bd1a44f0e4dd0d03e9a8/build.rs#L911-L919

Reply:
Applied